apps/docs/deploying-codemods/cli.mdx

```diff
@@ -72,10 +72,18 @@ Can be used to publish a codemod to Codemod Registry.
 
 Publishing codemods requires [logging in](#other-commands) to Codemod platform and having a codemod that is [compatible with Codemod Registry](/codemod-registry/introduction#required-codemod-package-structure).
 
+To do so, use the `publish` command inside the codemod package directory:
+
 ```bash
 codemod publish
 ```
 
+Alternatively, you can publish a codemod from a local <Tooltip tip="This file can be a simple transform file using any of the supported codemod engines.">transform file</Tooltip>:
+
+```bash
+codemod publish [path-to-file] --source
+```
+
```

Comment:
it can be not only a transform file, but also a codemod-compatible package that resides somewhere outside of user current directory. this is slightly misleading.